keadm/cmd/keadm/app/cmd/cloud/gettoken.go

```diff
@@ -17,14 +17,9 @@ import (
 
 var (
 	gettokenLongDescription = `
-"keadm gettoken" command prints the token to use for establishing bidirectional trust between edge nodes and cloudcore. 
-A token can be used when a edge node is about to join the cluster. With this token the cloudcore then approve the 
+"keadm gettoken" command prints the token to use for establishing bidirectional trust between edge nodes and cloudcore.
+A token can be used when a edge node is about to join the cluster. With this token the cloudcore then approve the
 certificate request.
-`
-	gettokenExample = `
```

Comment:
We shoud use the `gettokenExample` in L39

Reply:
Not sure, see https://github.com/kubeedge/kubeedge/blob/master/keadm/cmd/keadm/app/cmd/cloud/gettoken.go#L31-L51

Reply:
https://github.com/kubeedge/kubeedge/blob/master/keadm/cmd/keadm/app/cmd/cloud/init.go#L61

Reply:
It is `cloudInitExample` not `gettokenExample `.

Reply:
https://github.com/kubeedge/kubeedge/blob/master/keadm/cmd/keadm/app/cmd/cloud/gettoken.go#L39

Reply:
Get it.

Reply:
Please don't delete the gettokenExample and use it in L39